File: ontrack-git/src/main/java/net/nemerosa/ontrack/git/model/GitSynchronisationStatus.java
package net.nemerosa.ontrack.git.model;

/**
 * General status for the synchronisation.
 */
public enum GitSynchronisationStatus {

    /**
     * There is a repository
     */
    IDLE,

    /**
     * The synchronisation is currently running
     */
    RUNNING,

    /**
     * No synchronisation was ever done
     */
    NONE
}

File: ontrack-git/src/main/java/net/nemerosa/ontrack/git/model/GitSynchronisationStatus.kt
package net.nemerosa.ontrack.git.model

/**
 * General status for the synchronisation.
 */
enum class GitSynchronisationStatus {

    /**
     * There is a repository
     */
    IDLE,

    /**
     * The synchronisation is currently running
     */
    RUNNING,

    /**
     * No synchronisation was ever done
     */
    NONE
}
